package/gluon-ddhcpd/files/lib/gluon/upgrade/317-ddhcpd-disable

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+uci set ddhcpd.settings.enabled="0"
```

Comment:
This can be merged into `316-ddhcp`

Reply:
done, please check if it was done correctly